mig-api/api.go

```diff
@@ -70,6 +70,12 @@ func main() {
 	// unauthenticated endpoints
 	s.HandleFunc("/heartbeat", getHeartbeat).Methods("GET")
 	s.HandleFunc("/ip", getIP).Methods("GET")
+	// Loader manifest endpoints
+	s.HandleFunc("/manifest", authenticate(getManifest)).Methods("GET")
+	s.HandleFunc("/manifest/sign/", authenticate(signManifest)).Methods("POST")
+	s.HandleFunc("/manifest/status/", authenticate(statusManifest)).Methods("POST")
+	s.HandleFunc("/manifest/agent/", getAgentManifest).Methods("POST")
+	s.HandleFunc("/manifest/fetch/", getManifestFile).Methods("POST")
```

Comment:
if we're `GET`ing manifests, the methods should be `GET`, not `POST`. (POST is only used when some state on the server side is changed).